content/applications/inventory_and_mrp/purchase/manage_deals/calls_for_tenders.rst

```diff
@@ -0,0 +1,139 @@
+=================
+Calls for Tenders
+=================
+
+**Calls for Tenders** are one of the two main types of **Purchase Agreements** available by default
+in Odoo.
```

Comment:
The **Calls for Tenders** type of **Purchase Agreements** is available by default in Odoo.

(the initial wording seemed a bit clunky to me, so the above is just a suggestion. Nothing required - just a thought)